server/src/resolvers/mutation.js

```diff
@@ -27,37 +27,54 @@ module.exports.Mutation = {
   },
 
   signUpWithGoogle:
-      async (
-          _, {googleId, email, firstName, lastName, profileImgUrl},
-          {dataSources}) => {
-        const user = await dataSources.mainAPI.findOrCreateUser({googleId},
-            {
-              email,
-              firstName,
-              lastName,
-              profileImgUrl: profileImgUrl ? profileImgUrl : '',
-            });
-
-        if (user === null) {
-          return {
-            success: false,
-            message: cannotCreateUserMessage,
-            token: null,
-            user: null,
-          };
-        }
-
-        // TODO(lsh9034): Implement session logic.
-        // Create session and return sessionId to client.
+    async (
+      _, { googleId, email, firstName, lastName, profileImgUrl },
+      { dataSources }) => {
+      const user = await dataSources.mainAPI.findOrCreateUser({ googleId },
+        {
+          email,
+          firstName,
+          lastName,
+          profileImgUrl: profileImgUrl ? profileImgUrl : '',
+        });
 
+      if (user === null) {
         return {
-          success: true,
-          message: 'Success',
-          token: 'We have to implement this',
-          user,
+          success: false,
+          message: cannotCreateUserMessage,
+          token: null,
+          user: null,
         };
-      },
+      }
 
+      // TODO(lsh9034): Implement session logic.
+      // Create session and return sessionId to client.
```

Comment:
Is this another todo? if not, you don't have to put this with a new line. You can concatenate this with the above sentence.